src/api.ts

```diff
@@ -283,8 +283,21 @@ export interface ToolsRequest {
 }
 
 export type ToolsResponse = {
-  exchanges: Pick<Exchange | ExchangeAggregator, 'key' | 'name' | 'logoURI'>[]
-  bridges: Pick<Bridge, 'key' | 'name' | 'logoURI'>[]
+  exchanges: {
+    key: string
+    name: string
+    logoURI: string
+    supportedChains: ChainId[]
+  }[]
+  bridges: {
+    key: string
+    name: string
+    logoURI: string
+    supportedChains: {
+      fromChainId: ChainId
+      toChainId: ChainId
+    }[]
+  }[]
 }
```

Comment:
This is not needed anymore now, is it? also all other `{TYPE}Response` and `{TYPE}Request` could probably be removed?

Reply:
Yes, I was thinking about it but wasn't sure 😅 I will move all of them.